qp_shogun/utils.py

```diff
@@ -122,9 +122,9 @@ def _format_params(parameters, func_params):
         # Check for single letter commands
         if len(param) == 1:
             dash = '-'
-        if value is 'True':
```

Comment:
We had previously used strings instead of Bool here because at some point, we were generalizing the parameter formatting in such a way that parameters that got passed values like '1' would evaluate as 'True' and get just the `--param` flag rather than the `--param 1` flag/value pair. 

I *think* this is okay here, but do check to make sure that passing e.g. `--quality-cutoff 0` results in the formatted param `--quality-cutoff 15` rather than ``.